tests/integration/components/user-dropdown-test.js

```diff
@@ -1,10 +1,27 @@
 import { computed } from '@ember/object';
-import { run } from '@ember/runloop';
 import { moduleForComponent, test } from 'ember-qunit';
+import PageObject from 'ember-cli-page-object';
 import hbs from 'htmlbars-inline-precompile';
+import userDropdownComponent from 'code-corps-ember/tests/pages/component/user-dropdown';
+
+let page = PageObject.create(userDropdownComponent);
+
+function renderPage() {
+  page.render(hbs`
+{{user-dropdown
+  user=user
+  action='hide'
+   }}`);
+}
```

Comment:
You got some indentation crap here. You could probably fit it all onto a single line to.